test/e2e/specs/jetpack/plugins__install-remove.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @group jetpack
+ */
+
+import {
+	DataHelper,
+	TestAccount,
+	PluginsPage,
+	SnackbarNotificationComponent,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Jetpack: Plugin' ), function () {
+	const pluginName = 'Hello Dolly';
+	let page: Page;
+	let pluginsPage: PluginsPage;
+	let siteURL: string;
+
+	beforeAll( async function () {
+		page = await browser.newPage();
+
+		const testAccount = new TestAccount( 'jetpackUser' );
+		await testAccount.authenticate( page );
+		// This is bit of a hacky workaround to avoid committing
+		// in plain text the URL of our test sites.
+		// @TODO We really ought to make a function in DataHelper
+		// to somehow extract the user's site.
+		await page.waitForLoadState( 'networkidle' );
+		siteURL = page.url().split( 'stats/day/' ).slice( -1 )[ 0 ];
+	} );
+
+	it( 'Navigate to Plugins', async function () {
+		pluginsPage = new PluginsPage( page );
+		await pluginsPage.visit( siteURL );
+	} );
+
+	describe( 'Plugin: Install', function () {
+		beforeAll( async function () {
+			// Ensure known good state by removing the plugin
+			// if already installed.
+			await pluginsPage.visitPage( 'hello-dolly', siteURL );
+			if ( await pluginsPage.pluginIsInstalled() ) {
+				await pluginsPage.clickRemovePlugin();
+			}
```

Comment:
In the past we've shied away from writing tests that alter the state of the test account due to concerns when said tests are run in parallel. 

However, I believe this guard is sufficient for this spec as it is only ever run as part of the `@jetpack` test group, which itself is not scheduled regularly (at this time). Even when scheduled, I do not anticipate this spec to encounter a situation where parallel runs are taking place.